api/enterprise-api-ref/shared-groups.markdown

```diff
@@ -0,0 +1,720 @@
+                ]
+            }
+        }
+    ],
+    "meta": {
+        "count": 2,
+        "page": 1,
+        "timestamp": 1686739758,
+        "total": 2,
+        "hostsCountCacheTime": null
+    }
+}
+```
+
+
+## Make shared group personal
+
+**URI:** https://hub.cfengine.com/api/host-groups/shared/:id/makePersonal
+
+**Method:** POST
+
+**Parameters:**
+
+* **id** *(integer)*
+  Unique group identifier.
+
+**Example request:**
+
+```
+curl -k --user <username>:<password> \
+  -X POST \
+  https://hub.cfengine.com/api/host-groups/shared/5/makePersonal
+```
+
+**Example response:**
+
+API returns new ID of the personal group.
+
+```json
+{"id":"6"}
+```
+
+# Shared Groups CMDB
+
+The configuration management database (CMDB) API enables you to manage classes and variables for specific host groups.
+
+## List CMDB
+
+You can see a list of stored group-specific configurations
+
+**URI:** https://hub.cfengine.com/api/host-groups/shared/:id/cmdb
+
+**Method:** GET
+
+**Parameters:**
+
+* **id** *(integer)*
+  Unique group identifier.
+* **fromEpoch** *(integer)*
+  Returns configurations with epoch value greater than set in the filter.
+  Epoch is the sequence number of the latest CMDB change. In every API list request,
+  `cmdb_epoch` will be present in the meta section, which contains the maximum
+  epoch value among selected items. Optional parameter.
+* **fromTime** *(timestamp)*
+  Include changes performed within interval. Format: `YYYY-mm-dd HH:MM:SS` or `YYYY-mm-dd`. Optional parameter.
+* **toTime** *(timestamp)*
+  Include changes performed within interval. Format: `YYYY-mm-dd HH:MM:SS` or `YYYY-mm-dd`. Optional parameter.
+* **skip** *(integer)*
+  Number of results to skip for the processed
+  query. The Mission Portal uses this for pagination. Optional parameter.
+* **limit**  *(integer)*
+  Limit the number of results in the query. Optional parameter.
+
+**Example request (curl):**
+
+```console
+curl -k --user <username>:<password> \
+  -X GET \
+  https://hub.cfengine.com/api/host-groups/shared/4/cmdb?epochFrom=2
+```
+
+**Example response:**
+
+```
+HTTP 200 Ok
+{
+    "data": {
+        "5": {
+            "classes": {
+                "My_class": {},
+                "My_class2": {
+                    "comment": "comment helps to understand what this class serves for"
+                }
+            },
+            "variables": {
+                "HubCMDB:My.hostname": {
+                    "value": "host1.cfengine.com",
+                    "comment": "comment"
+                },
+                "Namespace:BundleName.VariableName": {
+                    "value": "myvalue"
+                }
+            }
+        }
+    },
+    "meta": {
+        "count": "1",
+        "page": 1,
+        "timestamp": 1619116399,
+        "total": "1",
+        "cmdb_epoch": "13"
+    }
+}
+```
+
+## Get group's specific configuration
+
+**URI:** https://hub.cfengine.com/api/host-groups/shared/:id/cmdb/:type/:name/
```

Comment:
Is it expected for this trailing slash to be strictly required?

I was struggling with the API for a while until I finally realized I was missing the trailing `/` things worked as soon as I added it.